pkg/common/utils/netaddr_test.go

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2022 CloudWeGo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import "testing"
+
+func TestNetAddr(t *testing.T) {
+	networkAddr := NewNetAddr("127.0.0.1", "127.0.0.1")
```

Comment:
Better use different two strings, because it is difficult to tell if functionality is both ok.

Reply:
i think i fixed it now.
Let me know if i am wrong.

Reply:
Because both network and address are set to "127.0.0.1", so from the assertion bellow, there exist the possibility that the `networkAddr.Network()` get the value from address and `networkAddr.String()` get the value from network.
It will still pass the test.
By the way, it is a small logical loophole in this Test case.

Reply:
I changed the both values to different address and network.
https://github.com/cloudwego/hertz/pull/372/files#diff-ca8e1d9b2c7fd974a7a2789ebda7a28f22c2317d0415fb87aa82e92c47563f4fR26-R30